ubuntudesign/asset-uploader
# Support multiple assets

Merge this script with changes from
https://gist.github.com/nottrobin/bad4d1b8f880bbb23ed10f5457d976bc.

This adds:

- Support for  uploading directories or multiple assets
- JSON output
- Better error messaging

QA
--

Try uploading multiple assets, using `url-path`, uploading all files in a directory, etc. And make sure you use the staging assets server.
